pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -1036,23 +1039,25 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		return
 	}
 	log.Infof("handleGlobalConfigModify for %s\n", key)
-	status := cast.CastGlobalConfig(statusArg)
-	var gcp *types.GlobalConfig
+	status := cast.CastConfigItemValueMap(statusArg)
+	var gcp *types.ConfigItemValueMap
 	debug, gcp = agentlog.HandleGlobalConfigNoDefault(ctx.subGlobalConfig,
```

Comment:
Why do we need the NoDefault part??

Reply:
@kalyan-nidumolu I had to add that because of the prevalence of setting the default LogLevel to Debug making logmanager run the system out of memory. Hence you can only get logmanager to run with Debug if you explicitly set it.